fury/material.py

```diff
@@ -200,3 +201,61 @@ def _create_points_material(
         )
     else:
         raise ValueError(f"Unsupported material type: {material}")
+
+
+def _create_text_material(
+    *,
+    color=(0, 0, 0),
+    opacity=1.0,
+    outline_color=(0, 0, 0),
+    outline_thickness=0.0,
+    weight_offset=1.0,
+    aa=True,
```

Comment:
`aliasing=True` instead

Reply:
Hello Serge, 
if we gonna change it to aliasing, we gonna get the error below.
 TypeError: Material.__init__() got an unexpected keyword argument 'aliasing'
as per pygfx, for Textmaterial function they used argument aa.
https://github.com/pygfx/pygfx/blob/main/pygfx/materials/_text.py

Reply:
Hi @ManishReddyR what you need to do is to change the variable name on our side. And keep the same in theirs.


```suggestion
    aliasing=True,
```